includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -1338,8 +1341,34 @@ private function is_valid_layout( $tag_spec, $node ) {
 			return true;
 		}
 
-		// Elements without a width or height don't need to be validated.
-		if ( ! $node->hasAttribute( 'width' ) && ! $node->hasAttribute( 'height' ) ) {
+		$layout_attr  = $node->getAttribute( Attribute::LAYOUT );
+		$input_width  = new CssLength( $node->getAttribute( Attribute::WIDTH ) );
+		$input_height = new CssLength( $node->getAttribute( Attribute::HEIGHT ) );
+
+		if ( ! $input_width->isDefined() && ! $input_height->isDefined() ) {
+			/*
+			 * Special case. If no layout related attributes were provided, this implies
+			 * the CONTAINER layout. However, telling the user that the implied layout
+			 * is unsupported for this tag is confusing if all they need is to provide
+			 * width and height in, for example, the common case of creating
+			 * an AMP-IMG without specifying dimensions. In this case, we emit a
+			 * less correct, but simpler error message that could be more useful to
+			 * the average user.
+			 *
+			 * See https://github.com/ampproject/amp-wp/issues/4465
+			 */
+			if (
+				'' === $layout_attr
```

Comment:
It seems somewhat more robust to check if this is falsy rather than a strict empty string. Some implementations of `getAttribute` return an empty string but others return `null`. See https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#Non-existing_attributes

```suggestion
				! $layout_attr
```

Reply:
Applied in 433b4bd5c206402e617a18ddd7059a1ba0ac4e86.